providers/standard/tests/unit/standard/decorators/test_python_virtualenv.py

```diff
@@ -41,6 +43,10 @@
 _Invalid = Any
 
 
+@pytest.mark.skipif(
+    not AIRFLOW_V_3_0_PLUS,
+    reason="Decorators were part of core not providers, so this test doesnt make sense for < AF3.",
+)
```

Comment:
Just want to highlight this for others.

The reason we did this, is that On Airflow 2.x, `task.python` and all the other previous "core" operators were hard-coded in https://github.com/apache/airflow/blob/2.10.5/airflow/decorators/__init__.py#L61-L71

Which means that it is impossible for users to use these decorators on Airflow 2, meaning we should skip these tests in the compat tests, as the decorators specifically won't ever be using the one from the standard provider, even if installed.

Reply:
Resolving this comment, but yes thats right